README.md

```diff
@@ -1,55 +1,82 @@
 # cfgov-refresh
 
-This repository contains the redesign-in-progress of consumerfinance.gov. This includes front-end assets and build tools, and configuration for [Sheer](https://github.com/cfpb/sheer) to load content from Wordpress and Django back-ends to elasticsearch to render the site.
+The redesign-in-progress of the [consumerfinance.gov](http://consumerfinance.gov) website.
+This includes front-end assets and build tools,
+and configuration for [Sheer](https://github.com/cfpb/sheer)
+to load content from WordPress and Django back-ends to Elasticsearch to render the site.
 
-### This project is a work in progress
+**Technology stack**:
+- Mac OSX.
+- [Homebrew](http://brew.sh) - package manager for installing system software on Mac OSX.
+- Python and PIP (Python package installer).
+- WordPress API data source URL.
 
-![screen shot 2015-01-28 at 10 47 12 am](https://cloud.githubusercontent.com/assets/235397/5940816/3529246a-a6db-11e4-9bfc-76d5b5220a36.png)
+**This project is a work in progress.**
+Nothing presented in this repo—whether in the source code, issue tracker,
+or wiki—is a final product unless it is marked as such or appears on www.consumerfinance.gov.
```

Comment:
"www." is used here but not at L:3. This should be consistent for a more unified voice.

*There isn't an official guide but apparently we generally don't include the "www"